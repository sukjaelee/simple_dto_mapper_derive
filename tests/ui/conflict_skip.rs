use simple_dto_mapper_derive::DtoFrom;

mod types {
    pub struct Source {
        pub id: String,
    }
}

#[derive(DtoFrom)]
#[dto(from = types::Source)]
struct Dto {
    #[dto(skip, into)]
    id: String,
}

fn main() {}
