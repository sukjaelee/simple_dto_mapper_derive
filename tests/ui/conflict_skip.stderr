error: expected identifier, found `::`
 --> tests/ui/conflict_skip.rs:9:10
  |
9 | #[derive(DtoFrom)]
  |          ^^^^^^^
  |          |
  |          expected identifier
  |          while parsing this struct
  |
  = note: this error originates in the derive macro `DtoFrom` (in Nightly builds, run with -Z macro-backtrace for more info)

error: proc-macro derive produced unparsable tokens
 --> tests/ui/conflict_skip.rs:9:10
  |
9 | #[derive(DtoFrom)]
  |          ^^^^^^^

error[E0063]: missing field `id` in initializer of `Dto`
 --> tests/ui/conflict_skip.rs:9:10
  |
9 | #[derive(DtoFrom)]
  |          ^^^^^^^ missing `id`
  |
  = note: this error originates in the derive macro `DtoFrom` (in Nightly builds, run with -Z macro-backtrace for more info)
